### DISCUSSION TITLE
Valoracion de Practica

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-Sensores-y-Actuadores/practica-de-la-1er-semana-grupo-4#25] [Announcements]
Nota Grupal: 6.5 (Siete)

Gisela Romero: @GiseRomero  Aspiradora IoT: 6(Seis)
Federico Birge: @FedeBirge Aspiradora IoT: 6( Seis )
Carlos Javier Carunchio: @LitoCarunchio HidrateSpark: 7 ( Siete )
Maria Luciana Fereira: @marlufer SmartBand: 6 ( Seis )
Ema Vilma Gutierrez: @Emygut Monitoreo de glucosa: 6.5 (Siete). No realizo diagrama interno y presento .doc
Esteban Carrizo: @estebancarrizo Wereable Canino: 6.5 ( Siete ). No realizo diagrama interno y presento .doc

***
Marcos Nicolodi @marcosnicolodi: NC no completo.

**Comment by FedeBirge** [comment]
Gracias profe por la devolución.  Mientras nos corregía le dábamos la edición final, ya que nos retrasamos por el compañero que no sigue como le comenté en la clase, y tenía tarea asignada.
